lib/src/api/engine/remote/ws/native.rs

```diff
@@ -410,6 +508,28 @@ impl Response {
 	fn try_from(message: &Message) -> Result<Option<Self>> {
 		match message {
 			Message::Text(text) => {
+				// Live queries currently don't support the binary protocol
+				// This is a workaround until live queries add support to send messages over binary
+				if let Ok(Value::Object(Object(mut response))) = crate::sql::json(text) {
+					if let Some(Value::Object(Object(mut map))) = response.remove("result") {
+						if let Some(Value::Uuid(id)) = map.remove("id") {
+							if let Some(value) = map.remove("action") {
+								if let Ok(action) = from_value(value) {
+									if let Some(result) = map.remove("result") {
+										return Ok(Some(Self {
+											id: None,
+											result: Ok(Data::Live(Notification {
+												id,
+												action,
+												result,
+											})),
+										}));
+									}
+								}
+							}
+						}
+					}
+				}
```

Comment:
Hey @rushmorem what is the reason that the binary protocol can't support live notifications at this point in time?

Reply:
Hey @tobiemh, it's not that it can't support live queries. It's just that I noticed that currently live query notifications are sent as JSON. I haven't dug much into it yet, I was planning to rectify this in a follow up PR but doing it in this PR is fine with me too. My suspicion is that this happens because JSON is the default protocol and currently the binary protocol is selected depending on whether the client sent the request as a binary message. Since a live query notification doesn't originate from the client, it currently falls back to the default.

Reply:
Fixed.